krkn/scenario_plugins/pvc/pvc_scenario_plugin.py

```diff
@@ -313,5 +333,56 @@ def to_kbytes(self, value):
         res = int(value[:-2]) * (base**exp)
         return res
 
+    @staticmethod
+    def rollback_temp_file(
+        rollback_content: RollbackContent,
+        lib_telemetry: KrknTelemetryOpenshift,
+    ):
+        """Rollback function to remove temporary file created during the PVC scenario.
+
+        :param rollback_content: Rollback content containing namespace and encoded rollback data in resource_identifier.
+        :param lib_telemetry: Instance of KrknTelemetryOpenshift for Kubernetes operations.
+        """
+        try:
+            namespace = rollback_content.namespace
+            
+            # Decode rollback data from resource_identifier
+            rollback_data = json.loads(rollback_content.resource_identifier)
```

Comment:
think the issue is here, the script that gets created doesn't `import json`
<img width="1006" height="607" alt="Screenshot 2025-11-13 at 4 07 22 PM" src="https://github.com/user-attachments/assets/7d527387-bf2b-481b-b25b-10b0294e7f5b" />